### DISCUSSION TITLE
Sound in webview pauses background music e.g. Spotify (only on iOS)

### DISCUSSION THREAD

**Discussion by AlbinNewSeed** [react-native-webview/react-native-webview#3204] [Q&A]
If I have spotify music playing in background and a sound is played in the webview, then the spotify music is paused. (on iOS)
It doesn't pause on Android. 

If anyone can point me in the right direction of what I would need to modify in my app to fix this, it would be much appreciated! 

Thanks!

**Comment by Azeem00786** [comment]
I am using an MP4 video for my app's splash screen. Some clients want to play music, so I have enabled audio. However, when my audio starts playing, the background music stops automatically. Is there any solution available for this issue?

**Comment by 8128** [comment]
I'm having the same issue on this - you cannot keep other audios playing if you started watching a video from webview. If I have Youtube app playing video in the background, it will be paused as well. I think this might be an interrupt mode problem. Can we have an option to switch from things like `Do not mix`, `Duck others`, `Mix with others`...

This only happens to android devices, in iOS you can still have audio in the background.

**Reply by 8128** [reply]
Yesterday, I stumbled upon an intriguing observation: when a video player within a webview starts playing, it pauses any background audio. However, if the webview contains both a playing audio (which must be actively playing) and a video player, initiating playback of the video doesn't halt the background audio. This results in three concurrent audio streams: the audio within the webview, the video's audio within the webview, and the background audio. A crude workaround for this issue involves substituting the webview audio with a silent looped audio, effectively circumventing the background audio interruption. It's merely a makeshift solution, albeit an unconventional one.
